src/app/components/task/GeolocationRespondTask.js

```diff
@@ -331,9 +332,9 @@ class GeolocationRespondTask extends Component {
           onBlur={() => this.setState({ openResultsPopup: false })}
           fullWidth
         />
-        <div style={{ font: caption, color: black54 }}>
+        <Box color={black54} fontFamily={fontStackSans} fontWeight={400} fontSize={units(1.5)} lineHeight={units(2.5)}>
```

Comment:
Please revert. This is better handled with the material-ui `Typography` component.